pyproject.toml

```diff
@@ -64,6 +64,7 @@ dependencies = [
   "psutil",
   "pycryptodome", # used in updated/casync, panda, body, and a test
   "setproctitle",
+  "xattr",
```

Comment:
You will need to run `uv lock` and commit the new `uv.lock` to make the tests runs